test/integration/integration_test_fixtures.py

```diff
@@ -178,6 +212,23 @@ def getTestServerRootUri(self):
     return super(HttpIntegrationTestBase, self).getTestServerRootUri(False)
 
 
+class MultiServerHttpIntegrationTestBase(IntegrationTestBase):
+  """
+  Base for running plain http tests against multiple Nighthawk test servers
+  """
+
+  def __init__(self, ip_version, backend_count):
+    super(MultiServerHttpIntegrationTestBase, self).__init__(ip_version, backend_count)
+    self.nighthawk_test_config_path = os.path.join(
+        self.test_rundir, "test/integration/configurations/nighthawk_http_origin.yaml")
+
+  def getTestServerRootUri(self):
```

Comment:
Can we add doc comments for these? if there are just overrides for methods on the base class, we can just do:

```python
def getTestServerRootUri(self):
  """See base class."""
```

Of course we should ensure that the base class has documentation.

Reply:
Done